src/snowflake/snowpark/dataframe.py

```diff
@@ -3771,6 +3771,16 @@ def convert(col: ColumnOrName) -> Expression:
         exprs = [convert(col) for col in parse_positional_args_to_list(*cols)]
         return exprs
 
+    def printSchema(self) -> str:
```

Comment:
`printSchema` is supposed to print the schema, not necessarily returning a string. We can use https://stackoverflow.com/questions/20507601/writing-a-pytest-function-for-checking-the-output-on-console-stdout to test the output.

I'm also thinking to keep all the functions consistent in snowpark, we name this function `print_schema` and alias `printSchema` for compatibility